### PR TITLE
Add "Augmented/Mixed Reality (Passthrough)" section to "PWAs on Oculus Quest 2" blog post

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**

### CODE REVIEW

**Comment by FluorescentHallucinogen** [comment]
@tomayac PTAL. Could you please also help to upload the following video file to imgix?

https://user-images.githubusercontent.com/7892779/204152281-fcdc2863-c37b-4220-a606-cd6ec604670d.mp4

**Comment by tomayac** [comment]
Uploaded video:

```nunjucks
{% Video src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/RmPUmnh57gDZ2bbGkOWR.mp4" %}
```

File: src/site/content/en/blog/pwas-on-oculus-2/index.md [hunk 1]
```diff
@@ -514,6 +515,46 @@ from the Immersive Web Working Group's [WebXR Samples](https://immersive-web.git
 
 {% Video autoplay=true, muted=true, loop=true, playsinline=true, src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/nDm0rY5DvtyTbDgUqFTS.mp4" %}
 
+### Augmented/Mixed Reality (Passthrough)
+
+As announced at Meta Connect 2022, Meta Quest Browser
+[has added support](https://developer.oculus.com/documentation/web/webxr-mixed-reality/) for
+[WebXR Augmented Reality (AR)](/web-ar/), also known as Mixed Reality (MR), on Meta Quest 2 and Meta
```
Comment (tomayac) [inline src/site/content/en/blog/pwas-on-oculus-2/index.md:522]:
```suggestion
[WebXR Augmented Reality (AR)](/web-ar/), also known as Mixed Reality (MR), on Meta Quest&nbsp;2 and Meta
```

File: src/site/content/en/blog/pwas-on-oculus-2/index.md [hunk 2]
```diff
@@ -514,6 +515,46 @@ from the Immersive Web Working Group's [WebXR Samples](https://immersive-web.git
...
+        <a-sky color="#ECECEC" hide-on-enter-ar></a-sky>
+      </a-entity>
+    </a-scene>
+  </body>
+</html>
+```
+
+// TODO: add video
+
+Meta Quest 2 has monochrome cameras, so the passthrough is in grayscale, while Meta Quest Pro has
```
Comment (tomayac) [inline src/site/content/en/blog/pwas-on-oculus-2/index.md:555]:
```suggestion
Meta Quest&nbsp;2 has monochrome cameras, so the passthrough is in grayscale, while Meta Quest Pro has
```

File: src/site/content/en/blog/pwas-on-oculus-2/index.md [hunk 3]
```diff
@@ -514,6 +515,46 @@ from the Immersive Web Working Group's [WebXR Samples](https://immersive-web.git
...
+        <a-cylinder position="1 0.75 -3" radius="0.5" height="1.5" color="#FFC65D"></a-cylinder>
+        <a-plane position="0 0 -4" rotation="-90 0 0" width="4" height="4" color="#7BC8A4" hide-on-enter-ar></a-plane>
+        <a-sky color="#ECECEC" hide-on-enter-ar></a-sky>
+      </a-entity>
+    </a-scene>
+  </body>
+</html>
+```
+
+// TODO: add video
```
Comment (tomayac) [inline src/site/content/en/blog/pwas-on-oculus-2/index.md:553]:
```suggestion
// TODO: add video
```

Should we add the above demo as a Glitch?

**Review by tomayac — COMMENTED** [review]
Thanks for working on this! Some comments…

File: src/site/content/en/blog/pwas-on-oculus-2/index.md (hunk 3 above)
Comment (FluorescentHallucinogen) [reply src/site/content/en/blog/pwas-on-oculus-2/index.md]: What do you mean? It's already here: https://github.com/GoogleChrome/web.dev/pull/9125/files#diff-c6a7fb61774789f401f0c1ce34ce6702e4e2e3e718f1ef862a2e45d881f80147R525

Comment (tomayac) [reply src/site/content/en/blog/pwas-on-oculus-2/index.md]: You can embed Glitches: https://web.dev/handbook/web-dev-components/#embed-a-glitch.

**Comment by FluorescentHallucinogen** [comment]
@tomayac Why the Netlify bot doesn't generate preview builds (like https://github.com/GoogleChrome/web.dev/pull/9121#issuecomment-1327632809) for my pull requests?

Since [Glitches](https://web.dev/handbook/web-dev-components/#embed-a-glitch) are actually iframes, I need to test the blog post page right on my Meta Quest 2 headset to figure out which permission policies to fill in the `allow` array. Preview builds would be very useful for this.

File: src/site/content/en/blog/pwas-on-oculus-2/index.md (hunk 3 above)
Comment (FluorescentHallucinogen) [reply src/site/content/en/blog/pwas-on-oculus-2/index.md]: Good idea!

Comment (FluorescentHallucinogen) [reply src/site/content/en/blog/pwas-on-oculus-2/index.md]: @tomayac Done. :wink:

**Review by tomayac — APPROVED** [review]
LGTM from my end. 

Over to @rachelandrew fir final approval. 

@FluorescentHallucinogen, please check if the permissions are propagated properly once the staging link is up. If not, you may need to use a raw iframe embed, and not the widget.

**Comment by FluorescentHallucinogen** [comment]
@tomayac
> please check if the permissions are propagated properly once the staging link is up. If not, you may need to use a raw iframe embed, and not the widget.

I've deployed the preview build at https://web-dev-my-staging.web.app/pwas-on-oculus-2/ and checked. The raw iframe embed is not needed. All works like a charm with the widget. :)

**Comment by FluorescentHallucinogen** [comment]
@rachelandrew Friendly ping. :wink:

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**